### DISCUSSION TITLE
Bitmaps and the File Manager

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1515] [Q&A] [answered]
Bitmaps and the File Manager

How do I make the File Manager notice and track bitmaps? Some programs such as [Spy embed bitmaps in the code](https://github.com/Interlisp/medley/blob/54b26070700511727b0a20a9a3f93866cc037493/library/SPY#L87) but I haven't figured how to do that.

**Comment by nbriggs** [comment, marked as answer]
If you have it in a variable it just goes as a regular VARS in the filecoms.

**Reply by pamoroso** [reply]
It works great, thanks.

**Comment by nbriggs** [comment]
and if it's more complicated (such as a circular structure) see the `HORRIBLEVARS` file package command.

**Comment by pamoroso** [comment]
More specifically I do something like this:

```
← (SETQ MYICON (EDITBM))
← (ADDTOFILE `(MYICON ,MYICON) 'VARS 'FILENAME)
```
